packages/cashier/src/types/stores/client-store.types.ts

```diff
@@ -41,6 +41,7 @@ export type TClientStore = {
         decimal_places?: number;
     };
     loginid?: string;
+    mt5_login_list: Array<DetailsOfEachMT5Loginid>;
```

Comment:
```suggestion
    mt5_login_list: DetailsOfEachMT5Loginid[];
```
Looks more readable this way